golang/btcspv/bitcoin_spv_test.go

```diff
@@ -10,651 +9,577 @@ import (
+	ExtractOutputScriptLen       []tutils.ExtractOutputScriptLenTC     `json:"extractOutputScriptLen"`
+	ExtractHash                  []tutils.ExtractHashTC                `json:"extractHash"`
+	ExtractHashError             []tutils.ExtractHashError             `json:"extractHashError"`
+	ExtractValue                 []tutils.ExtractValueTC               `json:"extractValue"`
+	ExtractValueLE               []tutils.ExtractValueLETC             `json:"extractValueLE"`
+	ExtractOpReturnData          []tutils.ExtractOpReturnDataTC        `json:"extractOpReturnData"`
+	ExtractOpReturnDataError     []tutils.ExtractOpReturnDataError     `json:"extractOpReturnDataError"`
+	ExtractInputAtIndex          []tutils.ExtractInputAtIndexTC        `json:"extractInputAtIndex"`
+	ExtractInputAtIndexError     []tutils.ExtractInputAtIndexError     `json:"extractInputAtIndexError"`
+	IsLegacyInput                []tutils.IsLegacyInputTC              `json:"isLegacyInput"`
+	DetermineInputLength         []tutils.DetermineInputLengthTC       `json:"determineInputLength"`
+	ExtractScriptSig             []tutils.ExtractScriptSigTC           `json:"extractScriptSig"`
+	ExtractScriptSigError        []tutils.ExtractScriptSigError        `json:"extractScriptSigError"`
+	ExtractScriptSigLen          []tutils.ExtractScriptSigLenTC        `json:"extractScriptSigLen"`
+	ValidateVin                  []tutils.ValidateVinTC                `json:"validateVin"`
+	ValidateVout                 []tutils.ValidateVoutTC               `json:"validateVout"`
+	ExtractInputTxIDLE           []tutils.ExtractInputTxIDLETC         `json:"extractInputTxIdLE"`
+	ExtractTxIndexLE             []tutils.ExtractTxIndexLETC           `json:"extractTxIndexLE"`
+	ExtractTxIndex               []tutils.ExtractTxIndexTC             `json:"extractTxIndex"`
+	DetermineOutputLength        []tutils.DetermineOutputLengthTC      `json:"determineOutputLength"`
+	DetermineOutputLengthError   []tutils.DetermineOutputLengthError   `json:"determineOutputLengthError"`
+	ExtractOutputAtIndex         []tutils.ExtractOutputAtIndexTC       `json:"extractOutputAtIndex"`
+	ExtractOutputAtIndexError    []tutils.ExtractOutputAtIndexError    `json:"extractOutputAtIndexError"`
+	ExtractTarget                []tutils.ExtractTargetTC              `json:"extractTarget"`
+	ExtractTimestamp             []tutils.ExtractTimestampTC           `json:"extractTimestamp"`
+	Hash256MerkleStep            []tutils.Hash256MerkleStepTC          `json:"hash256MerkleStep"`
+	VerifyHash256Merkle          []tutils.VerifyHash256MerkleTC        `json:"verifyHash256Merkle"`
+	RetargetAlgorithm            []tutils.RetargetAlgorithmTC          `json:"retargetAlgorithm"`
+	CalculateDifficulty          []tutils.CalculateDifficultyTC        `json:"calculateDifficulty"`
+	Prove                        []tutils.ProveTC                      `json:"prove"`
+	CalculateTxId                []tutils.CalculateTxIDTC              `json:"calculateTxId"`
```

Comment:
nit: can this have a capital `ID` at the end?

Reply:
So, this is like this pretty much everywhere which is why I did it like this.  Should I go ahead and change this in all the languages?

Reply:
No, just in Go. I'm fine with it being lowercase elsewhere. but the go linter hates it

Reply:
Got it, this is already fixed then.  Should be good to merge